core/statistics.c

```diff
@@ -376,6 +376,8 @@ volume_t *get_gas_used(struct dive *dive)
 		end = cyl->end.mbar ? cyl->end : cyl->sample_end;
 		if (end.mbar && start.mbar > end.mbar)
 			gases[idx].mliter = gas_volume(cyl, start) - gas_volume(cyl, end);
+		else
+			gases[idx].mliter = 0;
```

Comment:
I am not sure this is the correct thing to do. We simply don't know the value but that does not make it 0. Using zero when it was actually non-zero but unknown would result in a displayed SAC rate that is too small. We should rather compute the SAC rate only for those parts of the dive where a cylinder was breathed where we know how much.

Reply:
For example, I might be doing a dive with back gas (for which I have usage information) plus a deco gas (for which for example I forgot to memorise the initial pressure). Then I would like to get my SAC rate only for the part of the dive that used the back gas. And no value at all for the SAC rate if I had no usable gas information.

Reply:
Both callers of that function explicitly treat `mliter == 0` as "don't consider this cylinder". If you think that a different mechanism should be used, feel free to submit a patch.

This PR just restores old behavior.

Reply:
I think that @bstoeger  is right. @atdotde can we close this and merge?

Reply:
ping @atdotde